File: ElectronicPrescription/Models/Prescription.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ElectronicPrescription.Models
{
    public class Prescription
    {
        public int PrescriptionId { get; set; }

        public DateTime ExpirationDate { get; set; }

        public int? MedicalReceiptId { get; set; }
        public MedicalReceipt MedicalReceipt { get; set; }

        public int? PresentationId { get; set; }

        public Presentation Presentation { get; set; }

        public Posology PrescribedPosology { get; set; }
    }
}

File: frontend-electronic-prescriptions/src/app/model/prescription.ts
import { Fill } from './fill'
import { Posology } from './posology'
import { Presentation } from './presentation'

export class Prescription {
    expirationDate: Date;
    quantity: number;
    presentation: Presentation;
    posology: Posology;
    medicine: string;
    fills: Fill[];
}